components/cam/bld/namelist_files/use_cases/2010_scream_hr.xml

```diff
@@ -145,7 +98,7 @@
 <drydep_list            >'H2O2', 'H2SO4', 'SO2'</drydep_list>
 
 <!-- sim_year used for CLM datasets and SST forcings -->
-<sim_year>2015</sim_year>
+<sim_year>2010</sim_year>
```

Comment:
Doesn't this mean we are going to use 2010 SST data?

Reply:
No, I think we still end up using this SST (sst_HadOIBl_bc_1x1_clim_c101029.nc).

Reply:
Also, changing this parameter came back with BFB results for ne4 tests run before and after this set of changes, which makes me think that setting sim_year has no impact on simulations. I wanted to set it to 2010 so that it's consistent with our forcing choices.